js/script.js

```diff
@@ -28,4 +37,62 @@ newGame.addEventListener("click", function() { // wywołanie funkcji
         }   else {
             printRounds(rounds);
         }
+});
+
+// COMPUTER CHOICE FUNCTION
+
+function getComputerChoice() { // Funkcja losowania "przycisku" przez komputer
+    var result = Math.floor(Math.random() * 3); // Funkcja losująca liczbę, mnożenie przez 3, zaokrąglenie w dół i zapis do zmiennej
+    if (result === 0) { // zamiana liczby na tekst
+        return "papier";
+    } else if (result === 1) {  // zamiana liczby na tekst
+        return "kamień";
+    }   else if (result === 2) {    // zamiana liczby na tekst
+        return "nożyce";
+    }
+};
+
+// PLAYER CHOICE FUNCTION
+
+function playerMove(playerChoice) {
+    var computerChoice = getComputerChoice(); // Zapisanie do zmiennej "gotowego" wyboru komputera
+
+    if (playerChoice === computerChoice) { // Porównaie wyników
+        result.innerHTML = "<strong>Remis.</strong> Gracz wybrał" + playerChoice + " - Komputer wybrał " + computerChoice;
+    } else if ((playerChoice === "papier" && computerChoice === "kamień") || (playerChoice === "kamień" && computerChoice === "nożyce") || (playerChoice === "nożyce" && computerChoice === "papier")) {
+            result.innerHTML = "<strong>Wygrana!</strong> Gracz wybrał " + playerChoice + " - Komputer wybrał " + computerChoice;
+            playerScore++;  // Zwiększenie liczby punktów gracza
+            userPoints.innerHTML = playerScore; // Wyświetlenie liczby punktów w tabeli wyników
+            // gameOver(); // Wywołanie funkcji kończącej grę
+        }   else {
+            result.innerHTML = "<strong>Przegrana.</strong> Gracz wybrał " + playerChoice + " - Komputer wybrał " + computerChoice;
+            computerScore++;    // Zwiększenie liczby punktów komputera
+            computerPoints.innerHTML = computerScore;   // Wyświetlenie liczby punktów w tabeli wyników
+            // gameOver(); // Wywołanie funkcji kończącej grę
+        }
+        gameOver();
+};
+
+// GAME OVER FUNCTION
+
+function gameOver() { // Funkcja kończąca grę
+    if (userScore === rounds) { // Porównanie liczby punktów gracza z ilością zadeklarowanych rund
+        gameResult.innerHTML = "Gracz wygrał tą rozgrywkę"; // Wyświetlenie komunikatu o wygranej gracza
+    }   else if (computerScore === rounds) {  // Porównanie liczby punktów komputera z ilością zadeklarowanych rund
+        gameResult.innerHTML = "Komputer wygrał tą rozgrywkę";  // Wyświetlenie komunikatu o wygranej komputera
+    }
+};
+
+// BUTTONS
+
+paperButton.addEventListener("click", function() {
+    playerMove("papier");
```

Comment:
zeby zablokowac gre przed nacisnieciem "nowa gra" moglbys dorobic jakas zmienna np `gamePossible` i na poczatku przypisac do niej `false`, a kiedy uzytkownik wprowadzi poprawna ilosc rund, przypisac jej `true`.
Wtedy w tym event listenerze mozesz sprawdzac, czy wartosc tej zmiennej jest `true` i jesli tak, to odpalasz `playerMove`, a jesli nie to wyswietlasz komunikat w stylu `nacisnij "nowa gra" aby rozpoczac`